src/targets/Logary.Targets.Stackdriver/Target_Stackdriver.fs

```diff
@@ -0,0 +1,173 @@
+﻿// DOCUMENT YOUR MODULE
+module Logary.Targets.Stackdriver
+
+open Hopac
+open Hopac.Infixes
+open Logary
+open Logary.Target
+open Logary.Internals
+open Google.Logging.V2
+open Google.Api
+open Google.Logging.Type
+open Logary.Utils.Chiron
+open Logary.Utils.Chiron.Builder
+
+/// A structure that enforces certain labeling invariants around monitored resources
+/// TODO: add more resource types.
+/// SEE: https://cloud.google.com/logging/docs/api/v2/resource-list#service-names
+/// SEE: https://cloud.google.com/logging/docs/migration-v2#monitored-resources
+type ResourceType = 
+  /// Compute resources have a zone and an instance id
+  | ComputeInstance of zone: string * instance : string
+  /// Containers have cluster/pod/namespace/instance data
+  | Container of clusterName : string * namespaceId : string * instanceId : string * podId : string * containerName : string * zone : string
+  static member createComputeInstance(zone, instance) = ComputeInstance(zone, instance)
+  static member createContainer(cluster, ns, instance, pod, container, zone) = Container(cluster, ns, instance, pod, container, zone)
+
+type StackdriverConf = 
+  {
+    /// Google Cloud Project Id
+    projectId : string
+    /// Name of the log to which to write
+    logId : string
+    /// The resource we are monitoring
+    resource : ResourceType
+    /// Any additional user labels to be added to the messages
+    labels : System.Collections.Generic.IDictionary<string,string>
+    /// how many messages to wait for before sending
```

Comment:
Technically it's the maximum number of messages to batch; any less and they'll get sent as a batch too.